### PR TITLE
Migrated CI and Vault to new setup

### PR DESCRIPTION
**Pull request by ericm**

### CODE REVIEW

File: roles/ci/tasks/main.yml [hunk 1]
```diff
@@ -2,30 +2,30 @@
 - name: Ensure Drone MySQL Database
   run_once: true
   mysql_db:
-    name: "{{ drone_mysql_database }}"
-    login_user: "{{ mysql_admin_username }}"
+    name: "{{ drone.mysql_database }}"
+    login_user: "{{ mysql_user }}"
```
Comment (ocanty) [inline roles/ci/tasks/main.yml:6]: what is mysql_user?

Comment (ericm) [reply roles/ci/tasks/main.yml:6]: root

Comment (ericm) [reply roles/ci/tasks/main.yml:6]: Used to setup drone's user etc

File: vars/mysql.yml [hunk 2]
```diff
@@ -1 +1,2 @@
-mysql: "{{ vault_mysql }}"
\ No newline at end of file
+mysql: "{{ vault_mysql }}"
+mysql_user: "root"
```
Comment (ocanty) [inline vars/mysql.yml:2]: probably not needed if mysql_* ansible modules operate like the postgres ones

File: roles/ci/tasks/main.yml (hunk 1 above)
Comment (ocanty) [reply roles/ci/tasks/main.yml:6]: meme

File: vars/mysql.yml (hunk 2 above)
Comment (ericm) [reply vars/mysql.yml:2]: apparently not? defaulting to netsoc

**Pull request merged**